python/test/test_ingest_stream.py

```diff
@@ -0,0 +1,106 @@
+import json
+
+from timeplus import Stream, Query
+import datetime
+
+
+def test_ingest(test_environment):
+    name = "test_ingest_stream"
+    data = ["time", "data"]
+    values = [[datetime.datetime.now(), "abcd"]]
+
+    stream = (
+        Stream(env=test_environment)
+        .name(name)
+        .column("time", "datetime64(3)")
+        .column("data", "string")
+        .create()
+    )
+
+    stream.ingest(data, values)
+
+    query_result = Query(env=test_environment).sql(f"select * from {name}").execute().fetchall()
```

Comment:
this is wrong

1. query has no execute and fetchall interface, that is from sqlalchamy
2. stream query wont give you result, as your query is for real-time data, wont return historical result